app/src/main/java/com/rkm/tasky/feature/task/screen/TaskScreen.kt

```diff
@@ -0,0 +1,279 @@
+    val deletedString = stringResource(R.string.item_detail_delete_success, stringResource(R.string.task))
+
+    val dropDownActions = ReminderDropDownMenuActions(
+        onExpanded = viewModel::onExpanded,
+        onSelectedDuration = viewModel::onSelectedDuration
+    )
+
+    val actions = ItemScreenActions(
+        onTimeClick = viewModel::showTimeDialog,
+        onDateClick = viewModel::showDateDialog,
+        onSaveClick = viewModel::onSave,
+        onEditClick = viewModel::onEdit,
+        onCloseClick = events.onNavigateBack,
+        onDeleteClick = viewModel::onDelete,
+        onEditField = events.onEditField,
+        updateDate = viewModel::updateDate,
+        updateTime = viewModel::updateTime
+    )
+    TaskScreen(
+        modifier = modifier,
+        task = task,
+        state = itemUiState,
+        dropDownMenuUiState = dropDownState,
+        actions = actions,
+        dropDownActions = dropDownActions
+    )
+
+    LaunchedEffect(
+        title
+    ) {
+        if(title != null) {
+            viewModel.onUpdateTitle(title)
+        }
+    }
+
+    LaunchedEffect(
+        desc
+    ) {
+        if(desc != null) {
+            viewModel.onUpdateDescription(desc)
+        }
+    }
+
+    ObserveAsEvents(viewModel.taskScreenEventChannel) { event ->
+        scope.launch {
+            when (event) {
+                is ItemUiEvent.ItemErrorEvent -> {
+                    val message = event.message.asString(context)
+                    SnackBarController.sendEvent(
+                        event = SnackBarEvent(
+                            message = message
+                        )
+                    )
+                }
+
+                ItemUiEvent.ItemCreateEvent -> {
+                    SnackBarController.sendEvent(
+                        event = SnackBarEvent(
+                            message = createdString
+                        )
+                    )
+                }
+
+                ItemUiEvent.ItemDeleteEvent -> {
+                    SnackBarController.sendEvent(
+                        event = SnackBarEvent(
+                            message = updatedString
+                        )
+                    )
+                }
+
+                ItemUiEvent.ItemUpdateEvent -> {
+                    SnackBarController.sendEvent(
+                        event = SnackBarEvent(
+                            message = deletedString
+                        )
+                    )
+                }
+            }
+        }
+    }
+}
+
+
+@Composable
+internal fun TaskScreen(
+    modifier: Modifier,
+    task: TaskUiModel,
+    state: ItemUiState,
+    dropDownMenuUiState: ReminderDropDownMenuUiState,
+    actions: ItemScreenActions,
+    dropDownActions: ReminderDropDownMenuActions
+) {
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .statusBarsPadding(),
```

Comment:
Aren't you using a Scaffold at the root? That should solve most window inset problems